docs/resources/Monster.md

```diff
@@ -0,0 +1,185 @@
+# Monster Resource
+
+Monsters ID's here.
+
+###### Monster IDs
+
+|           ID          | Hex |
+| ----------------------|-----|
+|  UNK                  | 00  |
+|  Rathian              | 01  |
+|  Fatalis              | 02  |
+|  Kelbi                | 03  |
+|  Mosswine             | 04  |
+|  Bullfango            | 05  |
+|  Yian Kut-Ku          | 06  |
+|  Lao-Shan Lung        | 07  |
+|  Cephadrome           | 08  |
+|  Felyne               | 09  |
+|  Veggie Elder         | 0A  |
+|  Rathalos             | 0B  |
+|  Aptonoth             | 0C  |
+|  Genprey              | 0D  |
+|  Diablos              | 0E  |
+|  Khezu                | 0F  |
+|  Velociprey           | 10  |
+|  Gravios              | 11  |
+|  Felyne?              | 12  |
+|  Vespoid              | 13  |
+|  Gypceros             | 14  |
+|  Plesioth             | 15  |
+|  Basarios             | 16  |
+|  Melynx               | 17  |
+|  Hornetaur            | 18  |
+|  Apceros              | 19  |
+|  Monoblos             | 1A  |
+|  Velocidrome          | 1B  |
+|  Gendrome             | 1C  |
+|  Rocks                | 1D  |
+|  Ioprey               | 1E  |
+|  Iodrome              | 1F  |
+|  Pugis                | 20  |
+|  Kirin                | 21  |
+|  Cephalos             | 22  |
+|  Giaprey / Giadrome   | 23  |
```

Comment:
I think this is wrong, other "preys" have separate ids, here it shouldn't be different

Reply:
I will also open a issue for this